modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/twostep/msg/GridH2QueryRequest.java

```diff
@@ -553,6 +557,25 @@ public GridH2QueryRequest runningQryId(Long runningQryId) {
         return this;
     }
 
+    /**
+     * @return Query label.
+     */
+    public String label() {
+        return label;
+    }
+
+    /**
+     * Sets query label.
+     *
+     * @param label Query label.
+     * @return {@code this} for chaining.
+     */
+    public GridH2QueryRequest label(String label) {
+        this.label = label;
```

Comment:
the same: plays with nullability, it can be nullable or not ?

Reply:
Added `@Nullable` to `GridH2QueryRequest` and `QueryDescriptor`